app/views/Project/EntryEdit/index.tsx

```diff
@@ -156,7 +158,9 @@ function EntryEdit(props: Props) {
     // ENTRY FORM
 
     // FIXME: set section initially
-    const [selectedSection, setSelectedSection] = useState<string | undefined>();
+    const [selectedSection, setSelectedSection] = useState<string | undefined>(
```

Comment:
We may need to make sure that the selected entryId is present on the list of entries

Reply:
Also, this holds information about selected section, not selected entry.